meta/runtime.yml

```diff
@@ -66,6 +66,8 @@ action_groups:
   - ec2_asg
   - ec2_asg_info
   - ec2_asg_scheduled_action
+  - ec2_asg_instance_refresh
+  - ec2_asg_instance_refreshes_info
```

Comment:
Minor naming point for consistency, I'm thinking that ec2_asg_instance_refresh_info would be better as it fits in more with the naming of other module pairs